camelot/view/action_steps/item_view.py

```diff
@@ -124,14 +136,23 @@ def update_table_view(table_view, step):
         list(model.add_columns(step['columns']))
         # filters can have default values, so they need to be set before
         # the value is set
-        table_view.set_filters([AdminRoute.action_for(tuple(action['route'])) for action in step['actions'] if action['render_hint'] in [RenderHint.COMBO_BOX.value, RenderHint.GROUP_BOX.value]])
+        for filter_state in step['filter_states']:
+            route = tuple(filter_state[0])
+            state = filter_state[1]
+            action = AdminRoute.action_for(route)
+            values = [mode['name'] for mode in state['modes'] if mode['checked']]
```

Comment:
volstaat het hier niet om gewoon de 'default' van de filter op te halen, en bij manier van spreken : 

model.set_filter(action, action.default)

Reply:
Sommige filters hebben nog extra logica in hun get_state.

De BankStatusFilter bijvoorbeeld heeft als default [All]. De get_state() resulteert echter in ['draft', 'complete', None].

Reply:
hmm, vervelend idd

nu als we toch voor alle actions die state er geaan mee in steken, maakt het ook weer nt uit om die state te gaan opbouwen